service-front/module/Application/config/module.config.php

```diff
@@ -594,10 +594,10 @@
             'TELEPHONE' => 'Telephone',
             'POST_OFFICE' => 'Post Office',
         ],
-        'identity_methods' => [
-            'nin' => 'National Insurance number',
-            'pn' => 'UK Passport (current or expired in the last 18 months)',
-            'dln' => 'UK driving licence (must be current) ',
+        'identity_labels' => [
+            'NATIONAL_INSURANCE_NUMBER' => 'National Insurance number',
+            'PASSPORT' => 'UK Passport (current or expired in the last 18 months)',
+            'DRIVING_LICENCE' => 'UK photocard driving licence (must be current) ',
         ],
```

Comment:
I think `DonorFlowController::donorIdCheckAction` needs to be updated since it still refers to identity_methods?

Or potentially that method and this config can be removed.